### ISSUE TITLE
catalog.py:73-76: Show root categories as matrix. We have...

### ISSUE THREAD

**Issue by 0pdd** [fidals/stroyprombeton#567]
The puzzle `556-3bf482e5` from #556 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/e85e582ea72260439fac277406a8553d27b1d6c9/stroyprombeton/views/catalog.py#L73-L76

The puzzle was created by duker33 on 07-Apr-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by duker33** [comment]
this is the list of categories, that'll bubble to the first level:
```
Бордюрный камень
Водоотводные сооружения
Водопропускные трубы: звенья, оголовки, откосные стенки, фундаменты
Водоснабжение и канализация
Гидротехническое строительство
Железнодорожные переезды
Железнодорожные платформы
Железобетонные изделия для укрепительных работ
Искусственные сооружения на специализированных автодорогах
Карнизные плиты
Козырьки
Колонны
Коммуникации
Конструкции фундаментов для металлических опор ЛЭП
Металлические конструкции непроходных кабельных эстакад. Серия 3.016.2-12, выпуск 2
Мосты и путепроводы
Надолбы и тумбы
Обстановка и отдельные элементы дороги
Опоры
Опоры ЛЭП
Перемычки и прогоны
Плиты бетонные тротуарные
Плиты дорожные
Плиты перекрытия ребристые
Подпорные стены
Путепроводы тоннельного типа
Ригели
Сваи
Сваи_
Стойки для опор контактной сети
Теплоснабжение и электроснабжение
Тротуарные и ограждающие блоки
Урны
Утяжелители болотные клиновидные. ТУ 102-421-86, "Проект 10412"
Утяжелители болотные (тип Аг) . НГ–796–61 (ЛЕНГИПРОИНЖПРОЕКТ)
Утяжелители охватывающие для магистральных трубопроводов (тип УБО). Проект 999Б
Утяжелители сборные кольцевые для магистральных трубопроводов (тип УТК). ТУ 102-264-81, "Проект №994"
Фундаменты и блоки
Фундаменты под опоры контактной сети для железных дорог
Фундаменты трансформаторов и подстанций
Цветочницы
Шахты лифтов
Элементы лестниц
Эстакады
```

**Comment by duker33** [comment]
i'll leave categories content as is. I'll just change some interface:
- show only second level categories on the /gbi page
- somehow don't show first level categories at the top menu dropdown
- rename top level categories throw admin UI. See renaming strategy below
- breadcrumbs will still contain current root categories

Renaming:

from|to
--|--
Дорожное строительство|Дороги
Строительство энергетических объектов|Энергетика
Общегражданское и промышленное строительство|Промышленность
Строительство инженерных сетей|Инженерные сети

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#581](https://github.com/fidals/stroyprombeton/issues/581), [#582](https://github.com/fidals/stroyprombeton/issues/582) are still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#593](https://github.com/fidals/stroyprombeton/issues/593) is still not solved; solved: [#581](https://github.com/fidals/stroyprombeton/issues/581), [#582](https://github.com/fidals/stroyprombeton/issues/582).
